samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -518,6 +548,19 @@ CONFIG_LED_CONTINUOUS_INDICATION - Continuous LED status indication
 CONFIG_TEST_COUNTER - Enable test counter
    Enables the test counter.
 
+.. _CONFIG_AT_CMD_REQUESTS:
+
+CONFIG_AT_CMD_REQUESTS - Enable AT command requests
+   AT command requests allow remote execution of modem AT commands, requested using application-specific AT commands.
+   See :ref:`nrf_cloud_mqtt_multi_service_remote_at` for details.
+
+.. _CONFIG_AT_CMD_REQUEST_RESPONSE_BUFFER_LENGTH:
+
+CONFIG_AT_CMD_REQUEST_RESPONSE_BUFFER_LENGTH - AT command request response buffer length (bytes)
```

Comment:
```suggestion
CONFIG_AT_CMD_REQUEST_RESPONSE_BUFFER_LENGTH - Buffer length of AT command request response (bytes)
```

Reply:
I can't do this word reorder because it changes the meaning -- It's the length of the request-response-buffer, rather than the buffer-length of the request response

Reply:
Okay, I misread it. Maybe have it as request-response buffer.

Reply:
I've rearranged to:

> Length of AT command request response-buffer (bytes)

So that it is extra clear